src/main/java/org/broadinstitute/hellbender/tools/spark/sv/AlignAssembledContigsSpark.java

```diff
@@ -81,7 +73,31 @@ protected void runTool(final JavaSparkContext ctx) {
             }
 
         });
-        assembledBreakpoints.saveAsTextFile(output);
+        allContigAlignments.saveAsTextFile(output);
+    }
+
+    /**
+     * Loads an RDD of {@link ContigsCollection} objects keyed by assembly ID from disk. The input file
+     * should be the output of as RunSGAViaProcessBuilderOnSpark.
+     */
+    static JavaPairRDD<String, ContigsCollection> loadContigsCollectionKeyedByAssemblyId(final JavaSparkContext ctx, final String inputPath) {
+        final JavaRDD<String> inputAssemblies = ctx.textFile(inputPath).cache();
```

Comment:
It can be evicted by other persistent data, but it's still occupying heap memory.  So that keeps us from re-using that space for user data or for RDD actions.  I could be all wet, but I think we probably need to unpersist.